src/renderers/webgl/WebGLTextures.js

```diff
@@ -12,7 +12,7 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 	//
 
-	var useOffscreenCanvas = typeof OffscreenCanvas !== 'undefined';
+	var useOffscreenCanvas = typeof document === 'undefined';
```

Comment:
Why are we changing this?

Reply:
`OffscreenCanvas` is also available in a non worker scenario. So testing for `document` seems more robust since it's definitely missing in a worker.

Reply:
I see... How about changing the name of the variable then?

```js
var isWorker = typeof document === 'undefined';
```

Reply:
Sure.

Reply:
Thanks!

However, my original question still stands. We are no longer going to use `OffscreenCanvas` for resizing in non-worker scenarios?

Reply:
I mean, the problem seems to be that ImageBitmap needs flipping, but I do not understand why this PR also reverts to not using OffscreenCanvas in non-worker scenarios.

Reply:
#15805 should enable texture resizing in workers. I though it's better to ensure the new code path is exclusively used in workers so the previous behavior of `resizeImage()` is not affected.

Well, maybe I'm too careful in that regard^^. I was just not 100% sure of side effects if the method always returns images of type `ImageBitmap` when `OffscreenCanvas` is available.

Reply:
I see. Yeah, lets not give up on `OffscreenCanvas` just yet. If we find problems with it we can report to browsers. Lets just flip the image for now.

Reply:
@mrdoob Ok, the latest commit switches back to OffscreenCanvas by default, and flips the texture in this case, which was the initial issue. Is that ok?